template/src/components/virtual-background/useVB.native.tsx

```diff
@@ -151,7 +146,18 @@ const VBProvider: React.FC = ({children}) => {
       default:
         break;
     }
-  }, [vbMode, selectedImage, saveVB, previewVideoTrack]);
+  }, [vbMode, selectedImage, saveVB, previewVideoTrack, isLocalVideoON]);
+
+  const applyVirtualBackgroundToMainView = async (
```

Comment:
is the subcomponent exported on customisation api `VideoPreview ` is it different from this view ?